app/components/collections/show/deposits_component.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= render Elements::Tables::TableComponent.new(id: 'deposits-table', classes: 'mb-5', label: 'Deposits for this collection', show_label: false) do |component| %>
-  <% component.with_header(headers: ['Deposit', 'Owner', 'Status', 'Modified', 'Link for sharing']) %>
+  <% component.with_header(headers: ['Deposit', 'Owner', 'Status', 'Modified', t('sharing_link.label')], each_tooltips: [nil, nil, nil, nil, t('sharing_link.tooltip_html')]) %>
```

Comment:
Beginning to wonder if we should consider table headers as their own objects (components, structs, hashes) that must have a label and may have a tooltip and one or more classes. But that's more food for thought for follow-up tickets than feedback on this PR.